stablehlo/reference/Element.h

```diff
@@ -27,7 +30,10 @@ class Element {
  public:
   /// \name Constructors
   /// @{
-  Element(Type type, Attribute value) : type_(type), value_(value) {}
+  Element(Type type, APInt val) : type_(type), value_(val) {}
+  Element(Type type, APFloat val) : type_(type), value_(val) {}
+  Element(Type type, std::complex<APFloat> val)
+      : type_(type), value_(std::make_pair(val.real(), val.imag())) {}
```

Comment:
How about renaming `val` to `value` to be consistent with `value_` and `getFooValue()`?